onnx2pytorch/convert/operations.py

```diff
@@ -1,181 +1,455 @@
+from collections import defaultdict
 from functools import partial
 
+import numpy as np
+import onnx
 import torch
 from torch import nn
 from torch.nn import functional as F
 from onnx import numpy_helper
+from torch.nn.modules.conv import _ConvNd
+from torch.nn.modules.linear import Identity
+from torch.nn.modules.pooling import _MaxPoolNd
+
 
 from onnx2pytorch.convert.attribute import extract_attributes
 from onnx2pytorch.convert.layer import (
     convert_layer,
     convert_linear_layer,
     convert_batch_norm_layer,
     convert_instance_norm_layer,
+    convert_lstm_layer,
 )
 from onnx2pytorch.operations import *
 from onnx2pytorch.operations.base import OperatorWrapper
 from onnx2pytorch.operations import Resize, Upsample
-from onnx2pytorch.utils import value_wrapper
+from onnx2pytorch.utils import (
+    get_inputs_names,
+    get_outputs_names,
+    value_wrapper,
+)
+
+
+class Loop(nn.Module):
```

Comment:
Wouldn't it make more sense to move Loop to operators/loop.py? Here doesn't seem like the right place.

Reply:
Done! (The imports in its `__init__` and `forward` are to avoid circular import errors.)